src/clj_kondo/impl/analyzer.clj

```diff
@@ -1896,11 +1896,14 @@
                         ([re-frame.core reg-event-db]
                          [re-frame.core reg-event-fx]
                          [re-frame.core reg-event-ctx]
-                         [re-frame.core reg-sub]
                          [re-frame.core reg-sub-raw]
                          [re-frame.core reg-fx]
                          [re-frame.core reg-cofx])
                         (re-frame/analyze-reg ctx expr (symbol (str resolved-namespace) (str resolved-name)))
+                        ([re-frame.core subscribe])
+                        (re-frame/analyze-subscribe ctx (next (:children expr)) (str resolved-namespace))
```

Comment:
We usually pass the entire expression to the analyzing function and then do the children thing over there. Can you adapt to that style?

Reply:
sure